sunpy/map/mapbase.py

```diff
@@ -134,19 +133,25 @@ def __init__(self, data, header, **kwargs):
         self._fix_naxis()
 
         # Setup some attributes
-        self._name = self.observatory + " " + str(self.measurement)
         self._nickname = self.detector
 
-        # Visualization attributes
-        self.cmap = cm.gray
-
         # Validate header
         # TODO: This should be a function of the header, not of the map
         self._validate()
-
-        # Set mpl.colors.Normalize instance for plot scaling
         self.mpl_color_normalizer = self._get_mpl_normalizer()
 
+        if self.dtype == np.uint8:
+            norm = None
+        else:
+            norm = self._get_mpl_normalizer()
```

Comment:
Well it gets overwritten by the sub-classes so each instruments chosen their own but there is a default now in case the class can use it (which a few can).